homeassistant/components/http.py

```diff
@@ -221,28 +220,6 @@ def send(self, request, filepath):
 _GZIP_FILE_SENDER = GzipFileSender()
 
 
-class HAStaticRoute(StaticRoute):
-    """StaticRoute with support for fingerprinting."""
-
-    def __init__(self, prefix, path):
-        """Initialize a static route with gzip and cache busting support."""
-        super().__init__(None, prefix, path)
-        self._file_sender = _GZIP_FILE_SENDER
-
-    def match(self, path):
-        """Match path to filename."""
-        if not path.startswith(self._prefix):
-            return None
-
-        # Extra sauce to remove fingerprinted resource names
```

Comment:
aiohttp_cors just got an update to 0.5 that makes it compatible 👍 

Where are we adding routes before starting ? We don't start the server until the homeassistant_start event is fired, which is after all components have set up ?